src/backend/distributed/commands/create_citus_local_table.c

```diff
@@ -217,13 +249,6 @@ ErrorIfUnsupportedCreateCitusLocalTable(Relation relation)
 	 */
 	ErrorIfRelationIsAKnownShard(relationId);
 
-	/*
```

Comment:
to me, instead of passing cascade to error function, it sounds a little more intuitive to follow https://github.com/citusdata/citus/pull/4462#discussion_r553756880